lib/model/migrations/20250113-01-disable-nullable-blob-content-types.js

```diff
@@ -0,0 +1,23 @@
+// Copyright 2025 ODK Central Developers
```

Comment:
This file is included for demo purposes - it will actually be included by https://github.com/getodk/central-backend/pull/1356